translations/ccs_household_gb_eng.pot

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
 "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2020-05-18 11:17+0100\n"
+"POT-Creation-Date: 2020-05-20 09:12+0100\n"
```

Comment:
As discussed before, POT files without any text changes shouldn't be checked in.